Client/Ecosia/Extensions/HomepageViewController+Ecosia.swift

```diff
@@ -82,6 +82,26 @@ extension HomepageViewController: NTPLibraryDelegate {
     }
 }
 
+extension HomepageViewController: NTPConfigurableNudgeCardCellDelegate {
+    func nudgeCardRequestToPerformAction(for cardType: HomepageSectionType) {
+        switch cardType {
+        case .newsletterCard:
+            BrazeService.shared.logCustomEvent(.newsletterCardClick)
```

Comment:
Conscious we got what could look like a Braze-specific event here. Though we'd add it as part of the `EngagementService`